tests/package.json

```diff
@@ -32,6 +32,7 @@
     "test-with-logs": "mocha --printlogs -r ts-node/register 'tests/**/test-*.ts'",
     "pre-build-contracts": "TS_NODE_TRANSPILE_ONLY=true ts-node ./pre-build-contracts.ts && npx prettier -w ./contracts/compiled/*.json",
     "test": "TS_NODE_TRANSPILE_ONLY=true mocha --parallel -r ts-node/register 'tests/**/test-*.ts'",
+    "para-test": "TS_NODE_TRANSPILE_ONLY=true mocha -r ts-node/register 'para-tests/**/test-*.ts'",
```

Comment:
It would be nice to create a script to retrieve the binary locally for relay rode, and call that script here.

And also maybe add a para-test-single :)

I can do this on a branch based on this one if you want

Reply:
Yes, you can move the binary retrieval (which is done in the CI) into a script (/scripts folder)